discovery-provider/src/api/v1/playlists.py

```diff
@@ -255,3 +257,70 @@ def get(self, playlist_id):
         users = get_reposters_for_playlist(args)
         users = list(map(extend_user, users))
         return success_response(users)
+
+full_trending_playlists_response = make_full_response("trending_playlists_response", full_ns, fields.List(fields.Nested(full_playlist_model)))
+
+full_trending_parser = reqparse.RequestParser()
+full_trending_parser.add_argument('time', required=False)
+full_trending_parser.add_argument('limit', required=False)
+full_trending_parser.add_argument('offset', required=False)
+full_trending_parser.add_argument('user_id', required=False)
+
+@full_ns.route("/trending")
```

Comment:
can we make a non-full one as well? just to include in public API?

Reply:
yeah can do